notebooks/signac-flow_Aggregate_Tutorial.ipynb

```diff
@@ -0,0 +1,219 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 3.3 signac-flow aggregation example\n",
+    "\n",
+    "## About\n",
+    "\n",
+    "This notebook contains a minimal example for running a signac-flow's **aggregation** feature which is not released yet.\n",
+    "The example demonstrates how to compare an ideal gas with a lennard jones fluid by calculating a p-V phase diagram.\n",
+    "\n",
+    "## Author\n",
+    "\n",
+    "Hardik Ojha\n",
+    "\n",
+    "## Before you start\n",
+    "\n",
+    "Make sure you installed signac. Since this work is under progress hence you'd need to clone the signac-flow repository from git into your system and install it in your virtual enviroment in editable mode.\n",
+    "You can follow these steps to install signac-flow:\n",
+    "\n",
+    "```\n",
+    "git clone https://github.com/glotzerlab/signac-flow.git\n",
+    "pip install -e signac-flow\n",
+    "cd signac-flow/\n",
+    "git checkout feature/introduce-aggregator-classes\n",
+    "```\n",
+    "\n",
+    "You can find the work done on the branch ``feature/introduce-aggregator-classes`` here: https://github.com/glotzerlab/signac-flow/pull/336"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import random\n",
+    "\n",
+    "import signac\n",
+    "from flow import FlowProject, Aggregate, make_aggregates\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "\n",
+    "# Enter the signac project directory\n",
+    "project = signac.init_project('AggregationTutorialProject', 'projects/tutorial-aggregation')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Creating an operation\n",
+    "\n",
+    "We aim to generate a plot of temperatures (in °C) v/s days of a month having 31 days. After that, we compare that plot with the average temperature of that month.\n",
+    "\n",
+    "Firstly, we will create an operation which is supposed to aggregate all jobs, using the ``Aggregate`` class, and plot the data, using ``matplotlib``. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class MyProject(FlowProject):\n",
+    "    pass\n",
+    "\n",
+    "@Aggregate(sort='day')\n",
+    "@MyProject.operation\n",
+    "def plot_average_temperature(*jobs):\n",
+    "    temp = [job.sp.temperature for job in jobs]\n",
```

Comment:
I would add comments or something in the text that explains how each of these decorators work